src/borg/archiver.py

```diff
@@ -1243,6 +1263,13 @@ def compare_or_defer(item1, item2):
                 assert hardlink_master_seen(item2)
                 compare_items(output, item1.path, item1, item2, hardlink_masters)
 
+            if args.json_lines:
+                print_output = print_json_output
+            else:
+                print_output = print_text_output
```

Comment:
```
print_output = print_json_output if args.json_lines else print_text_output
```

Reply:
ok. pythonic way is not 2nd nature to me yet.

Reply:
that's quite a nice space-saver (without being too cryptic), this I use it often.